### ISSUE TITLE
[IxMenuCategory] Allow surrounding nested `IxMenuItem` with `<a>` tags

### ISSUE THREAD

**Issue by robert-wettstaedt** [siemens/ix#686]
### Suggestion / feature request

I was playing around with the new [`IxMenuCategory`](https://ix.siemens.io/version-dev/docs/controls/navigation/vertical-tabs#category) because we have a use case for nested menu items. 

The way I have used the regular `IxMenu` before is by surrounding each `IxMenuItem` with an `<a>` tag because they are just links to pages in our apps, like this:

```jsx
<IxMenu>
  <a href="/">
    <IxMenuItem>
      Home
    </IxMenuItem>
  </a>
</IxMenu>
```

When doing this with the `IxMenuCategory`, the component will not recognize the `IxMenuItems` because they are not direct children. This means when expanding the UI element, the expanded region will not be big enough to show all of the children of `IxMenuCategory`. [This line](https://github.com/siemens/ix/blob/v2.0.0-beta.0/packages/core/src/components/menu-category/menu-category.tsx#L58C65-L58C65) in the `IxMenuCategory` source explicitly causes this behavior.

I know I could just use an `onClick` event handler on the `IxMenuItems` and manually navigate to the requested page. But I would argue that this is not good UX nor is it accessible.

So I suggest to support surrounding `IxMenuItems` with `<a>` tags (or other tags for that matter). If you do not agree, what is your recommended way of defining links in the menu?

Example: https://codesandbox.io/s/allow-surrounding-nested-ixmenuitem-with-a-tags-qty7xn

**Comment by danielleroux** [comment]
Should already work like you descibed it.
Just tested the your provided example and it works with `a href`.

With #689 we provided a fix to remove the default link styling.

**Comment by robert-wettstaedt** [comment]
@danielleroux I am not sure why it the provided example would work for you. This is how it looks for me:

<img width="240" alt="Screenshot 2023-08-15 at 08 32 09" src="https://github.com/siemens/ix/assets/3035576/d9d5566d-f6d8-40e1-a4c1-c9aa14225641">

Notice how the `Category with <a>` only shows one child element even though two elements are defined in the code.

**Comment by danielleroux** [comment]
If you check the development console the item is there, but the animation is calculated based on the wrong item height. This is fixed in #689 (see description of PR)

**Comment by robert-wettstaedt** [comment]
Yes this is exactly what I was describing in my initial request.
